apps/native-component-list/tsconfig.json

```diff
@@ -7,8 +7,8 @@
     "paths": {
       "@expo/vector-icons": ["react-native-vector-icons"]
     },
-    "types": ["jest"],
-    "typeRoots": ["./ts-declarations", "node_modules/@types"],
+    "types": ["jest", "node"],
```

Comment:
IIRC we do not use Node or Node scripts in any way in NCL, what was the problem/reason you decided to update TS config here?

Reply:
Without this `global` is not recognized 
<img width="671" alt="image" src="https://github.com/expo/expo/assets/11707729/a9aed9b3-4cba-4c4e-b96a-c516f35ceb36">

Reply:
For the `global` object, can you try `globalThis`? I think this should work.

Using `NodeJS.Timeout` seems kinda weird here 😅 It might be better to remove it.

Reply:
Yes, `NodeJS.Timeout` is probably incorrect in RN context, we have lately the similar problem on web, when the correct typing is `number` (this is a result of TS preferring Node global types when present over default context ones):
* https://stackoverflow.com/a/56970244/964692

For Next app we use the correct way which allow us to avoid any ignore comments was calling `setTimeout` directly from `window` context instead of relaying on on global resolve.

Reply:
Thanks for the suggestion @tsapeta, I was able to fix this for almost all cases but it did not work for `process.env` inside `html-elements` ([View.tsx](https://github.com/expo/expo/blob/main/packages/html-elements/src/primitives/View.tsx#L150)), any suggestions for this specific case?